pkg/routeagent_driver/handlers/calico/calico_suite_test.go

```diff
@@ -135,13 +161,40 @@ func (t *testDriver) getDefaultIPPoolIPIPMode() string {
 	return string(p.Spec.IPIPMode)
 }
 
+func (t *testDriver) getDefaultInstallationEncapsulation() string {
+
+	installation, err := calico.GetDefaultInstallation(t.dynClient, gvrInstallations)
+	Expect(err).To(Succeed())
```

Comment:
For sanity check to avoid potential panic:

```suggestion
	Expect(err).To(Succeed())
	Expect(installation.Spec.CalicoNetwork).NotTo(BeNil())
	Expect(installation.Spec.CalicoNetwork.IPPools).NotTo(BeEmpty())
```

Reply:
Added the extra checks.